src/components/action-popover/action-popover-menu.component.js

```diff
@@ -85,63 +94,71 @@ const ActionPopoverMenu = ({
     // so we use a smaller array which makes keyboard navigation easier as we don't have to filter it on every
     // keypress
```

Comment:
Seen as this comment ventures across two lines, would it be better to have the comment inside of /* */ instead?

Reply:
Yeah I'll update the comment to be a block before I squash and merge 👍

Reply:
Excellent.